app/views/admin/budget_investments/_investments.html.erb

```diff
@@ -55,26 +58,9 @@
         <%= investment.valuation_finished? ? t('shared.yes'): t('shared.no') %>
       </td>
       <td class="small">
-        <% if investment.selected? %>
-          <%= link_to toggle_selection_admin_budget_budget_investment_path(@budget,
-                                                                           investment,
-                                                                           filter: params[:filter],
-                                                                           page: params[:page]),
-                      method: :patch,
-                      remote: true,
-                      class: "button small expanded" do %>
-            <%= t("admin.budget_investments.index.selected") %>
-          <% end %>
-        <% elsif investment.feasible? && investment.valuation_finished? %>
-          <%= link_to toggle_selection_admin_budget_budget_investment_path(@budget,
-                                                                           investment,
-                                                                           filter: params[:filter],
-                                                                           page: params[:page]),
-                      method: :patch,
-                      remote: true,
-                      class: "button small hollow expanded" do %>
-            <%= t("admin.budget_investments.index.select") %>
-          <% end %>
+        <%= options = investment_selected_link_options(investment) %>
```

Comment:
This `<%=` should be `<%` to avoid printing the result of the function. We normally require screenshots to both improve the review and also to "force" the coder to manually verify interface changes just in case something happens...